test/snapshots/no-single-promise-in-promise-methods.mjs.md

```diff
@@ -1126,3 +1126,206 @@ Generated by [AVA](https://avajs.dev).
     Suggestion 2/2: Switch to \`Promise.resolve(…)\`.␊
       1 | Promise.resolve(null).then()␊
     `
+
+## invalid(1): Promise.all([promise])
```

Comment:
Tests above swithed `Promsie.all` to `Promise.race` to make sure the logic for other methods doesn't change.